.github/workflows/readme-update.yml

```
@@ -0,0 +1,25 @@
+  update-readme:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Rebuild README sections
+        uses: actions/github-script@v7
```

actionlint (1.7.12)

[error] missing input "script" which is required by action "actions/github-script@v7". all required inputs are "script"

(action)